src/pages/Quote/Quote.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Quote.css';
+
+const Quote = () => (
+  <div className="quoteContainer">
+    <h2 className="quoteheading">Welcome To Our Page!</h2>
+    <p className="quote">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    <p className="quote">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+  </div>
+);
```

Comment:
[OPTIONAL] You can give yourself a little challenge and try to create a mechanism that randomly displays a new quote every time I refresh the page. You can achieve that through dynamic rendering by creating a data file that contains an array of quotes, importing the data to your quotes page, then picking a random quote from the array to be displayed. 🤓